addons/binding/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/NetworkUtils.java

```diff
@@ -150,16 +155,18 @@
      * @param host The IP or hostname
      * @param port The tcp port. Must be not 0.
      * @param timeout Timeout in ms
-     * @param logger A slf4j logger instance to log IOException
      * @return
      * @throws IOException
      */
     public boolean servicePing(String host, int port, int timeout) throws IOException {
+        logger.trace("Prepare servicePing for {} on port: {}", host, port);
         SocketAddress socketAddress = new InetSocketAddress(host, port);
         try (Socket socket = new Socket()) {
             socket.connect(socketAddress, timeout);
+            logger.info("Sucessful socket connect for {} on port: {}", host, port);
```

Comment:
This should not be here, this will completely fill the logs.

See also @davidgraeff comment in another PR:

> Personally I would remove this logging line. Actually I tried to keep this utility class logger free on purpose. But if we agree on introducing a logger here, it should at least not be used in this hot code-path for a well-known output. For a local subnet scan for example, this method is called in parallel for every single valid IP in range and the logger is not a no-op.

https://github.com/openhab/openhab2-addons/pull/3503